scripts/import.md

```diff
@@ -0,0 +1,174 @@
+# Import script
```

Comment:
add a link back to the list of the scripts (something similar is done in each individual endpoint file toward the endpoint.md list)